e2e/test/prerequisites/run.csx

```diff
@@ -0,0 +1,81 @@
+#r "Newtonsoft.Json"
```

Comment:
Missing copyright header.
Also, is #r correct here?

Reply:
Yes, the #r is an Azure function thing